desktop/apps/artwork/components/auction_artworks/helpers.coffee

```diff
@@ -1,17 +1,29 @@
-_ = require 'underscore'
-masonry = require '../../../../components/artwork_masonry/index.coffee'
+masonry = require '../../../../components/artwork_masonry_4_column/index.coffee'
+moment = require 'moment'
+{ ARTWORK_DISPLAY_NUM } = require './config.coffee'
+{ include, shuffle, take } = require 'underscore'
 
-reduceArray = (artworks, rows) ->
-  elements = artworks.splice(0,3)
-  rows.push(elements)
+isLiveOpen = (status, live_start_at) ->
```

Comment:
@sweir27 - originally that's what I was using, but the GraphQL interface returned that it was deprecated and that I should use `status` instead. I could of course alias `status`, but wondering what you think?